src/routes/loginOAuth.js

```diff
@@ -40,4 +40,43 @@ server.post("/auth/login", async (req, res) => {
   }
 });
 
+server.post("/auth/changePassword", async (req, res) => {
+  const { username, oldPassword, newPassword } = req.body;
+  let encryptedPassword
+  await encrypt.encryptPassword(newPassword).then(encryptedPass => {
+    encryptedPassword = encryptedPass;
+  })
+
+  const user = await Users.findOne({ username });
+  if (!user) {
+    res.status(401).json({
```

Comment:
Now, you can return early the res.status here and the execution of your function will stop so you can get rid of the else statement down below.
```suggestion
    return res.status(401).json({
```